modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1790,20 +1792,30 @@ private void _deleteObjectEntry(
 			objectDefinition, serviceBuilderObjectEntry);
 		_checkRootDescendantNode(serviceBuilderObjectEntry, false);
 
-		if (!FeatureFlagManagerUtil.isEnabled("LPD-53981") ||
-			(serviceBuilderObjectEntry.getStatus() ==
-				WorkflowConstants.STATUS_IN_TRASH)) {
+		long groupId = serviceBuilderObjectEntry.getGroupId();
 
-			_objectEntryService.deleteObjectEntry(
-				serviceBuilderObjectEntry.getObjectEntryId());
```

Comment:
Hi @yuriymr, When the master syncs, could you also update the logic for the [ObjectEntryFolderResourceImpl](https://github.com/brianchandotcom/liferay-portal/blob/972ac041854a0e13019797155afc5f071ed90d86/modules/apps/headless/headless-object/headless-object-impl/src/main/java/com/liferay/headless/object/internal/resource/v1_0/ObjectEntryFolderResourceImpl.java#L432-L452) code.

cc @guilhermedcamacho

Reply:
The master is now synced.